firmware/application/apps/ui_mictx.cpp

```diff
@@ -590,6 +582,20 @@ MicTXView::MicTXView(
     // Trigger receiver to update modulation.
     if (rx_enabled)
         receiver_model.set_squelch_level(receiver_model.squelch_level());
+
+    radio::set_tx_max283x_iq_phase_calibration(iq_phase_calibration_value);
+    field_tx_iq_phase_cal.set_range(0, hackrf_r9 ? 63 : 31);  // max2839 has 6 bits [0..63],  max2837 has 5 bits [0..31]
+    field_tx_iq_phase_cal.on_change = [this](int32_t v) {
+        if (rx_enabled) {  // here it's necessary to stop and start audio if it was in use to adjust DMA and fix no audio output after iq modification
```

Comment:
Technically speaking, the problem is that the set_tx_max283x_iq_phase_calibration() is inadvertently changing the radio mode to "Standby".  (Please see PR #2008)